dascore/repr.css

```
@@ -0,0 +1,165 @@
+
+.dc-repr {
+  --dc-blue: #0969da;
+  --dc-bright_blue: #0a7ea4;
+  --dc-red: #b03050;
+  --dc-yellow: #9a5b00;
+  --dc-green: #0f7a6c;
+  --dc-cyan: #0e7490;
+  --dc-dark_orange: #bc4c00;
+  --dc-grey50: #6b7280;
+  --dc-rule: color-mix(in oklab, currentColor 22%, transparent);
+  font-family: var(--jp-code-font-family, ui-monospace, SFMono-Regular, Menlo, monospace);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)